loops_lab.js

```diff
@@ -0,0 +1,271 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
No major mistakes or errors. I think you demonstrate good understanding of usage and implementation of loops. Keep up the good work!